apidef/api_definitions.go

```diff
@@ -480,6 +480,8 @@ type Scopes struct {
 type APIDefinition struct {
 	Id                  ObjectId      `bson:"_id,omitempty" json:"id,omitempty" gorm:"primaryKey;column:_id"`
 	Name                string        `bson:"name" json:"name"`
+	Expiration          string        `bson:"expiration" json:"expiration"`
+	ExpirationTs        time.Time     `bson:"-" json:"-"`
```

Comment:
can we use `ExpirationTs` as `time.Time` pointer, so that we can exclude `Expiration`?

Reply:
When we use `time.Time`, it is marshalled as:
`"expiration": "2021-12-20T12:52:29.035661+03:00"`

but we are using a special format in old expiration as:
`"expiration": "2021-12-20 12:52"`

If we don't do that, we will break backward compatibility